pyts/multivariate/transformation/multivariate.py

```diff
@@ -83,7 +83,7 @@ def transform(self, X):
 
         Returns
         -------
-        X_new : array, shape = (n_samples, *) or (n_samples, n_features, *)
```

Comment:
It was not a mistake. The thing is that:
* `MultivariateTransformer` applies a transformer for univariate time series to each feature of a multivariate time series independently;
* The shape of the output depends on the shape of the output of the univariate transformer, which is what `*` is supposed to mean.

It is probably not that clear, and I know that it raises a warning when building the doc. Suggestions are welcomed, but the shape that you proposed are not correct.

Reply:
Why not give it a name like "output_dim" and explain it in the docstring ?

Reply:
Good idea!

Reply:
I put back the original docstring, I think it's better to keep this PR consistent with the title without touching a lot of modules and address the doc warnings when the new doc is released.